components/script/dom/element.rs

```diff
@@ -2259,15 +2259,19 @@ impl Element {
                 } else {
                     let win = self.owner_window();
                     let source = &**attr.value();
+                    let global = &self.owner_global();
                     // However, if the Should element's inline behavior be blocked by
                     // Content Security Policy? algorithm returns "Blocked" when executed
                     // upon the attribute's element, "style attribute", and the attribute's value,
                     // then the style rules defined in the attribute's value must not be applied to the element. [CSP]
-                    if doc.should_elements_inline_type_behavior_be_blocked(
-                        self,
-                        csp::InlineCheckType::StyleAttribute,
-                        source,
-                    ) == csp::CheckResult::Blocked
+                    if global
+                        .get_csp_list()
```

Comment:
This doesn't need to be fixed in this change, but `fn get_csp_list()` does not follow rust naming conventions, which we try to follow in Servo. (see https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter). It should just be `fn csp_list()`.